PKHeX.WinForms/Program.cs

```diff
@@ -93,6 +94,12 @@ private static void CurrentDomain_UnhandledException(object sender, UnhandledExc
                 // Todo: make this translatable
                 ErrorWindow.ShowErrorDialog("An unhandled exception has occurred.\nPKHeX must now close.", ex, false);
             }
+            catch (MissingMethodException)
+            {
+                // Todo: make this translatable
+                MessageBox.Show(".NET Framework 4.6 needs to be installed for this version of PKHeX to run.", "PKHeX Error", MessageBoxButtons.OK, MessageBoxIcon.Stop);
+                Process.Start(@"https://www.microsoft.com/en-us/download/details.aspx?id=48130");
```

Comment:
Time zones are weird.  I'm about to go to sleep, since it's 11:30PM.

Sort of a desperate attempt at giving user feedback, maybe if the application's underlying console was shown, with a message to get the correct framework version.  It would involve referencing kernal32, which should be independant from the framework.  Hopefully the Console class still works.

Simply call ConsoleManager.Show, then use the Console clas to write to it.  (C# conversion is easy, and I link to the source in the comments, which may still be valid):
https://github.com/evandixon/SkyEditor.Core/blob/master/SkyEditor.Core.Windows/Windows/Internal/ConsoleManager.vb